django/db/backends/sqlite3/creation.py

```diff
@@ -101,3 +110,17 @@ def test_db_signature(self):
         else:
             sig.append(test_database_name)
         return tuple(sig)
+
+    def setup_worker_connection(self, _worker_id):
+        alias = self.connection.alias
+        worker_db = f'file:memorydb_{str(alias)}_{str(_worker_id)}?mode=memory&cache=shared'
+        sourcedb = sqlite3.connect(f'file:{str(alias)}_{str(_worker_id)}.sqlite3', uri=True)
```

Comment:
The naming is inconsistent `worker_db`, `sourcedb`, `second_db`, also f-strings should not contain function calls. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). IMO, it's more readable with %-formatting:
```suggestion
        worker_db = 'file:memorydb_%s_%s?mode=memory&cache=shared' % (alias, _worker_id)
        source_db = sqlite3.connect('file:%s_%s.sqlite3' % (alias, _worker_id), uri=True)
```

I would try to reuse `get_test_db_clone_settings()` :thinking:

Reply:
Maybe I'm missing something, but why do these need to be wrapped with `str()` anyway? You could do `{alias!s}` instead of `{str(alias)}`, but if it isn't required, just use `{alias}`. In that case you can use f-strings.